.github/workflows/vs2022-build.yml

```
@@ -0,0 +1,37 @@
+  TESTING:
+    name: MSVC 2022 Build (${{ matrix.platform }}-${{ matrix.config }})
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        config: [Debug, Release]
+        platform: [x64, Win32]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)